### DISCUSSION TITLE
Build books with unified/MDX (compatible with existing GitBooks)

### DISCUSSION THREAD

**Discussion by MatthewCaseres** [unifiedjs/unified#111] [Show and tell]
I can put GitBooks on my own website without making a fork or any modifications whatsoever. Unified is super useful for me!

[Check it out!](https://www.openedtech.dev/Open-EdTech/next-mdx-books/about.md)

I am using unified's parsers and utilities to parse markdown configuration files from GitBook and put them into JSON with some extra metadata.

We support: 
* Content is entirely separate from source code, mdx is loaded via next-mdx-remote from a repository containing only markdown.
* No modifications required to existing GitBook repositories.
* Adding a new book is as simple as adding a URL to a configuration.
* load files with or without h2 tags in the side navigation (compatibility with BOTH legacy and modern style GitBooks)
*  loading files locally (for developtment) or from raw.githubusercontent.com
* provide links to source code on GitHub from each page

Plans to build a library of educational components and use overridden code blocks with YAML configurations within.

**Comment by wooorm** [comment]
Nice!
The links here is broken btw
But very interesting ✨

**Reply by MatthewCaseres** [reply]
Eek! Thanks, fixed.
